composer.json

```diff
@@ -1,32 +1,42 @@
 {
-  "name": "cwp/cwp-installer",
-  "description": "CWP Project Template",
-  "license": "BSD-3-Clause",
-  "require": {
-    "cwp/cwp-recipe-basic": "1.8.x-dev",
-    "cwp/cwp-recipe-blog": "1.8.x-dev",
-    "cwp/starter-theme": "1.1.x-dev"
-  },
-  "require-dev": {
-    "cwp/cwp-recipe-basic-dev": "^1.1@stable"
-  },
-  "suggest": {
-    "silverstripe/textextraction": "Required for document text searching within CWP"
-  },
-  "config": {
-    "process-timeout": 900
-  },
-  "extra": {
-    "branch-alias": {
-      "dev-master": "1.8.x-dev"
-    }
-  },
-  "repositories": [
-    {
-      "type": "composer",
-      "url": "https://packages.cwp.govt.nz/"
-    }
-  ],
-  "prefer-stable": true,
-  "minimum-stability": "dev"
+    "name": "cwp/cwp-installer",
+    "type": "silverstripe-recipe",
+    "description": "CWP Project Template",
+    "license": "BSD-3-Clause",
+    "require": {
+        "php": ">=5.6.0",
+        "silverstripe/recipe-plugin": "^1",
+        "cwp/cwp-recipe-core": "2.x-dev",
+        "cwp/cwp-recipe-cms": "2.x-dev",
+        "silverstripe/recipe-blog": "1.x-dev",
+        "silverstripe/recipe-form-building": "1.x-dev",
+        "silverstripe/recipe-authoring-tools": "1.x-dev",
+        "silverstripe/recipe-collaboration": "1.x-dev",
+        "silverstripe/recipe-reporting-tools": "1.x-dev",
+        "cwp/cwp-recipe-search": "1.x-dev",
```

Comment:
cwp recipes were to start at 2.x as we discussed earlier.

Reply:
recipe-search master is aliased as 2.x-dev anyway.

Reply:
Yep, that's my mistake!